python/src/robyn/data/entities/calibration_input.py

```diff
@@ -47,13 +41,26 @@ class CalibrationInput:
     CalibrationInput is an immutable data class that holds the necessary inputs for a calibration process.
 
     Attributes:
-        channel_data (Dict[str, ChannelCalibrationData]): Dictionary with channel names as keys and ChannelCalibrationData instances as values.
+        channel_data: Dictionary mapping channel identifiers to their calibration data.
+            Keys can be either strings or tuples of strings for combined channels.
     """
 
-    channel_data: Dict[str, ChannelCalibrationData] = field(default_factory=dict)
+    channel_data: Dict[Union[str, Tuple[str, ...]], ChannelCalibrationData] = field(default_factory=dict)
+
+    def __post_init__(self):
+        # Convert string keys with '+' to tuples if needed
+        new_channel_data = {}
+        for key, value in self.channel_data.items():
+            if isinstance(key, str) and "+" in key:
+                new_key = tuple(key.split("+"))
```

Comment:
we shouldn't support option of combination of channel with concatenated string with "+" at all.